docs/Course Wiki/ECE Course Offerings/ECE120.md

```diff
@@ -0,0 +1,55 @@
+
+## Life After
+
+After completing ECE120, ECE students are encouraged to take ECE220, the second core computer engineering course, as soon as possible. ECE220 builds onto concepts introduced at the tail end of ECE120, which in turn build on concepts covered throughout the rest of the course; therefore, having a strong understanding of ECE120 material is important prior to taking ECE220. In due time after taking the core ECE courses, ECE385 offers a continuation of logical design elements, as well as hardware and software implementations of applicable scenarios. ECE411 offers an even more in-depth continuation of the computer organization concepts covered in ECE120 and ECE385, and is generally taken by students in their senior year after completing the rest of the computer engineering core classes.
+
+## Infamous Topics
+
+- Sequential logic design: This is covered about halfway through the course, and alongside combinational design forms the backbone of every computing device imaginable. The idea of sequential design centers around feeding combinational gate outputs back into each other and observing changes over time with a varying clock. This can be a bit obtuse to understand, but is vital to know for designing finite state machines, memories, and eventually a full CPU, which the class builds up to in the coming weeks.
+- Finite state machines: Covered soon after sequential logic design, finite state machines (FSMs) are the core of a computing device as they dictate what decision to make given a current state of operation and inputs. Finite state machine design is often diffucult for students to understand and apply to given scenarios, made even harder if they don't properly understand sequential logic, which is what FSMs build upon. This is why, for good reason, many students consider finite state machines to be one of the hardest, if not the hardest part of the course.
+
+## Resources
+
+Apart from the class wiki page, the [Introduction to Computing Systems: from Bits & Gates to C & Beyond](http://highered.mheducation.com/sites/0072467509/index.html) (2nd edition) textbook written by Patt and Patel offers an expansive detail of the coursework and includes some homework problems that additionally improve understanding and preparation for testing. Course notes, prepared by Professor Lumetta, are available at the ECE Supply Center and [online](https://wiki.illinois.edu/wiki/download/attachments/728159173/ece120-fall-2020-notes-for-students.pdf?version=1&modificationDate=1598033876080&). Other available resources include office hours for material mastery and clarification, and an open lab that allows students to finish lab work.
```

Comment:
specifically 2nd edition?

Reply:
I took that part from the old course wiki, which listed it as such, so I kept it in.